doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+accomplish this, we use the ``Client.put_tensor()`` function with the respective
+database client instances.
+
+For the single-sharded database:
+
+.. code-block:: python
+
+  array_1 = np.array([1, 2, 3, 4])
+  driver_client_single_shard.put_tensor("tensor_1", array_1)
+
+For the multi-sharded database:
+
+.. code-block:: python
+
+  array_2 = np.array([5, 6, 7, 8])
+  driver_client_multi_shard.put_tensor("tensor_2", array_2)
+
+Lets check to make sure the database tensors do not exist in the incorrect databases:
+
+.. code-block:: python
+
+  check_single_shard_db_tensor_incorrect = driver_client_single_shard.key_exists("tensor_2")
+  check_multi_shard_db_tensor_incorrect = driver_client_multi_shard.key_exists("tensor_1")
+  logger.info(f"The multi shard array key exists in the incorrect database: {check_single_shard_db_tensor_incorrect}")
+  logger.info(f"The single shard array key exists in the incorrect database: {check_multi_shard_db_tensor_incorrect}")
+
+When you run the experiment, the following output will appear:
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####] INFO The multi shard array key exists in the incorrect database: False
+  00:00:00 system.host.com SmartSim[#####] INFO The single shard array key exists in the incorrect database: False
+
+Initializing the colo Model
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In the next stage of the experiment, we will utilize the
+application script by configuring and creating
+a SmartSim colocated ``Model``.
+
+Step 1: Configure
+"""""""""""""""""
+You can specify the run settings of a model.
+In this example, we invoke the Python interpreter to run a
+python script. To specify this, we use the ``Experiment.create_run_settings()`` function.
+This function returns a ``RunSettings`` object.
+Here, we specify the path to the source code file (script) to
+``exe_args`` and the run command to ``exe``.
+
+.. code-block:: python
+
+  model_settings = exp.create_run_settings(exe="python", exe_args="./model_multidb_example.py")
+
+With the ``RunSettings`` instance,
+configure the the distribution of computational tasks (``RunSettings.set_nodes()``) and the number of instances
+the script is execute on each node (``RunSettings.set_tasks_per_node()``). In this
+example, we specify to SmartSim that we intend to execute the script once on a single node.
+
+.. code-block:: python
+
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+
+Step 2: Initialize
+""""""""""""""""""
+Next, create a ``Model`` instance using the ``Experiment.create_model()``.
+Pass the ``model_settings`` object as an argument
+to the ``create_model()`` function and assign to the variable ``model``.
+
+.. code-block:: python
+
+  model = exp.create_model("colocated_model", model_settings)
+
+Step 2: Colocate
+""""""""""""""""
+To colocate the model, use the ``Model.colocate_db_tcp()`` function to
+Colocate an Orchestrator instance with this Model over TCP/IP.
```

Comment:
Let's make this UDS, since that's what we recommend most users to use

Reply:
Will do